Added "Learning the elixir language" ebook (#6573)
EbookFoundation/free-programming-books: books/free-programming-books-langs.md

* [Getting Started Guide](http://elixir-lang.org/getting-started/introduction.html) (HTML) [(PDF, MOBI, EPUB)](https://github.com/potatogopher/elixir-getting-started)
* [Joy of Elixir](https://joyofelixir.com) - Ryan Bigg (HTML) - [Source](https://github.com/radar/joyofelixir) (:construction: *in process*)
* [Learning Elixir](http://learningelixir.joekain.com) - Joseph Kain Blog (HTML)

> * [Learning the Elixir Language](https://riptutorial.com/ebook/elixir) - Created by Stack Overflow Users (PDF) 

Reviewer: ```suggestion
* [Learning the Elixir Language](https://riptutorial.com/Download/elixir-language.pdf) - Compiled from StackOverflow Documentation (PDF) 
```
